charts/tf-controller/templates/deployment.yaml

```diff
@@ -25,10 +25,10 @@ spec:
       containers:
       - args:
         - --watch-all-namespaces
-        - --log-level=info
+        - --log-level={{ .Values.logLevel }}
         - --log-encoding=json
         - --enable-leader-election
-        - --concurrent=1
+        - --concurrent={{ .Values.concurrency | toString }}
```

Comment:
I think it should be {{ .Values.concurrency | quote }} if you want to make it a quoted string.